src/constants/chains.ts

```diff
@@ -1,103 +1,158 @@
-import { DAPP_URL } from "./env";
+import { Chain, Chains, CosmosChainID, EVMChainID } from "types/chain";
 
-export enum chainIDs {
-  junoMain = "juno-1",
-  junoTest = "uni-6",
-  polygonMain = "137",
-  polygonTest = "80001",
-  polygonLocal = "1337",
-  binanceMain = "56",
-  binanceTest = "97",
-  ethMain = "1",
-  ethTest = "5",
-  terraMain = "phoenix-1",
-  terraTest = "pisco-1",
-  //add axelar, connext
-}
-
-type Info = { txExplorer: string; addressExplorer: string; name: string };
-
-const explorers: { [key in chainIDs]: string } = {
-  1: "https://etherscan.io",
-  5: "https://goerli.etherscan.io",
-  97: "https://testnet.bscscan.com",
-  56: "https://bscscan.com",
-  137: "https://polygonscan.com",
-  80001: "https://mumbai.polygonscan.com",
-  1337: "",
-  "juno-1": "https://www.mintscan.io/juno",
-  "uni-6": "https://testnet.mintscan.io/juno-testnet",
-  "phoenix-1": "https://finder.terra.money/mainnet",
-  "pisco-1": "https://finder.terra.money/testnet",
+//mainnets
+export const polygon: Chain = {
+  id: "137",
+  brand: "polygon",
+  name: "Polygon",
+  rpc: "https://polygon.llamarpc.com",
+  lcd: "",
+  blockExplorer: "https://polygonscan.com",
+  nativeToken: {
+    id: "137",
+    symbol: "MATIC",
+    decimals: 18,
+    coinGeckoId: "matic-network",
+  },
 };
 
-const _chains: { [key in chainIDs]: Info } = {
-  [chainIDs.ethMain]: {
-    name: "Ethereum",
-    txExplorer: `${explorers[chainIDs.ethMain]}/tx`,
-    addressExplorer: `${explorers[chainIDs.ethMain]}/address`,
-  },
-  [chainIDs.ethTest]: {
-    name: "Ethereum",
-    txExplorer: `${explorers[chainIDs.ethTest]}/tx`,
-    addressExplorer: `${explorers[chainIDs.ethTest]}/address`,
-  },
-  [chainIDs.binanceMain]: {
-    name: "Binance",
-    txExplorer: `${explorers[chainIDs.binanceMain]}/tx`,
-    addressExplorer: `${explorers[chainIDs.binanceMain]}/address`,
-  },
-  [chainIDs.binanceTest]: {
-    name: "Binance",
-    txExplorer: `${explorers[chainIDs.binanceTest]}/tx`,
-    addressExplorer: `${explorers[chainIDs.binanceTest]}/address`,
+export const ethereum: Chain = {
```

Comment:
add  more data to already hardcoded chains so to not fetch chain record anymore from AWS. 
`rpc`, `lcd` and `blockExplorer` doesn't need to match from what's in AWS